unicodetools/src/main/java/org/unicode/props/IndexUnicodeProperties.java

```diff
@@ -10,7 +10,6 @@
 import com.ibm.icu.text.Normalizer2;
```

Comment:
The main reason for having special values for code point and name was to reduce the size of the data in memory. ( I think the .bin files are also zipped so at least the name part probably isn't so bad on disk). 

That probably isn't a big issue having (eg) 80000 extra name strings at this point, so I'm ok with these changes.

Reply:
I think the size of the data in memory remains the same; getUnicodeMap only transiently creates the full map.

Reply:
Right. It is a different issue (performance) if the rebuilding is repeated a lot during normal usage.